tests/unit/proxy/roundtrip/roundtrip_gremlin_proxy.py

```diff
@@ -0,0 +1,250 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+from datetime import datetime
+from typing import Any, List
+
+from amundsen_common.models.table import (Application, Column,
+                                          ProgrammaticDescription, Table)
+from amundsen_common.models.user import User
+from amundsen_gremlin.gremlin_model import EdgeTypes, VertexTypes
+from amundsen_gremlin.gremlin_shared import (make_cluster_uri, make_column_uri,
+                                             make_database_uri,
+                                             make_description_uri,
+                                             make_schema_uri, make_table_uri)
+from gremlin_python.process.traversal import Direction
+from overrides import overrides
+
+from metadata_service.proxy.gremlin_proxy import (_V, AMUNDSEN_TIMESTAMP_KEY,
+                                                  ExecuteQuery, FromResultSet,
+                                                  GenericGremlinProxy,
+                                                  _expire_other_links, _link,
+                                                  _properties_except,
+                                                  _properties_of, _upsert,
+                                                  timestamp)
+from metadata_service.proxy.statsd_utilities import timer_with_counter
+
+from .roundtrip_base_proxy import RoundtripBaseProxy
+
+LOGGER = logging.getLogger(__name__)
+
+
+class RoundtripGremlinProxy(GenericGremlinProxy, RoundtripBaseProxy):
```

Comment:
thanks, we could file a ticket to ask for help and see if the community could help to implement the round trip test for neo4j/atlas.

Reply:
Sounds great! I'd definitely suggest taking the approach we did with Neptune -- move the neo4j publishing code from databuilder into a shared lib so we can use the same code both in databuilder to load data in production and in metadata service to validate read/write in the roundtrip tests.